helpers/ComponentsHelper.js

```diff
@@ -40,7 +40,8 @@ export const withAppModels = (Component, AppModelProviders) => function WithAppM
   }, null);
 };
 
-export default {
+const exports = {
   withViewModel,
   withAppModels
 };
+export default exports;
```

Comment:
Why did you rewrite it this way?

Reply:
ESLint was failing here

Reply:
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-anonymous-default-export.md